github/code-scanning.go

```diff
@@ -18,11 +18,17 @@ import (
 // GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/code-scanning/
 type CodeScanningService service
 
+type Tool struct {
```

Comment:
```suggestion

// Tool represents the tool used to generate a Github Code Scanning Alert
//
// GitHub API docs: https://docs.github.com/en/rest/reference/code-scanning#list-code-scanning-alerts-for-a-repository
type Tool struct {
```

Reply:
Thank you, @stephengroat !

Following idiomatic Go style, please end line 21 with a period to make it a complete sentence for the auto-generated godocs.  The same goes for line 27 below.